src/App.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import App from './App';
+
+const setupBeforeAddTask = () => {
+  const utils = render(<App />);
```

Comment:
`utils`보다는 조금 더 명확하게 `renderApp` 은 어떤가요?

그리고 구조분해 할당으로 받아와 처리해 주는것도 좋을 것 같아요~
```suggestion
  const { getByLabelText, getByText } = render(<App />);
```